docs/csharp/language-reference/keywords/init.md

```diff
@@ -10,17 +10,29 @@ helpviewer_keywords:
 ---
 # init (C# Reference)
 
-In C# 9 and later, the `init` keyword defines an *accessor* method in a property or indexer. An init-only setter assigns a value to the property or the indexer element only during object construction. For more information and examples, see [Properties](../../programming-guide/classes-and-structs/properties.md), [Auto-Implemented Properties](../../programming-guide/classes-and-structs/auto-implemented-properties.md), and [Indexers](../../programming-guide/indexers/index.md).
+In C# 9 and later, the `init` keyword defines an *accessor* method in a property or indexer. An init-only setter assigns a value to the property or the indexer element **only** during object construction. This is useful as a kind of runtime constant, so that it can only be set once in the constructor, and never be changed again.
```

Comment:
I would change the wording for consistency:

```suggestion
In C# 9 and later, the `init` keyword defines an *accessor* method in a property or indexer. An init-only setter assigns a value to the property or the indexer element **only** during object construction. This enforces immutability, so  that once the object is initialized, it can't be changed again.
```